samcli/local/apigw/local_apigw_service.py

```diff
@@ -110,6 +110,8 @@ def create(self):
             static_folder=self.static_dir,  # Serve static files from this directory
         )
 
+        self._app.url_map.strict_slashes = False
```

Comment:
Can we  add a comment on what this means.

Reply:
Sure can do.